rct229/rulesets/ashrae9012019/ruleset_functions/get_hvac_systems_serving_zone_health_safety_vent_reqs_test.py

```diff
@@ -0,0 +1,98 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_hvac_systems_serving_zone_health_safety_vent_reqs import (
+    get_hvac_systems_serving_zone_health_safety_vent_reqs,
+)
+from rct229.schema.schema_utils import quantify_rmr
+from rct229.schema.validate import schema_validate_rmr
+
```

Comment:
Should there be tests that there are no typos in `LIGHTING_SPACE_TYPES_MATCH_REQ` and `VENTILATION_SPACE_TYPES_MATCH_REQ`?  I guess that would amount to checking that none of the values are None.

Reply:
Correct.
System 4 built into the Test RMI is used for test when both `ventilation_space_type` and `lighting_space_type` are not provided in a space.
I do not think Typos could exist because we run schema validation before running rule checkings and typo would be picked up by schema validation due to mismatch enum.

Reply:
I was referring to possible typos in your code where you list the `LIGHTING_SPACE_TYPES_MATCH_REQ` options. I believe if you typed one of the enumerations incorrectly, its value would be None.